CHANGELOG.md

```diff
@@ -14,6 +14,26 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [3.0.1] - 2019-04-26
+### Added
+- Code formatting tools for pom.xml and java source files
+- Support for use with Configuration as Code plugin, with test dependencies on configuration-as-code
+  version 1.9
+- Secret used during integration test can be provided through GOOGLE_CREDENTIALS_FILE environment
+  variable which should be a path to the secret file
+- Documentation transferred from wiki.jenkins.io and stored in source
+
+### Changed
+- Use unique IDs for each instance
+- Test classes run in parallel using the concurrency property in pom.xml
```

Comment:
Was this test classes in general or more specifically integration tests?

Reply:
It actually applies to the unit tests as well. it probably accounts for maybe 30 seconds of speedup but still good to have.